test/calibration/test_calibrations.py

```diff
@@ -1352,3 +1355,29 @@ def test_alternate_date_formats(self):
         self.addCleanup(self._remove_files, self._prefix)
         self.cals._params = defaultdict(list)
         self.cals.load_parameter_values(self._prefix + "parameter_values.csv")
+
+    def test_save_load_library(self):
+        """Test that we can load and save a library.
+
+        These libraries contain both parameters with schedules and parameters without
+        any schedules (e.g. frequencies for qubits and readout).
+        """
+
+        library = FixedFrequencyTransmon()
+        backend = FakeArmonk()
+        cals = BackendCalibrations(backend, library)
+
+        cals.parameters_table()
+
+        cals.save(file_type="csv", overwrite=True, file_prefix=self._prefix)
+
+        cals.load_parameter_values(self._prefix + "parameter_values.csv")
+
+        # Test the value of a few loaded params.
+        self.assertEqual(cals.get_parameter_value("amp", (0,), "x"), 0.5)
+        self.assertEqual(
+            cals.get_parameter_value("qubit_lo_freq", (0,)),
+            backend.defaults().qubit_freq_est[0],
+        )
+
+        self.addCleanup(self._remove_files, self._prefix)
```

Comment:
This should be the first line after save() so that files are always cleaned up even if the test encounters an error.

More generally, since all the test methods in this class use this same `addCleanup` call, it should probably just go in a  `tearDown` method (with `_remove_files` made robust to the case where the files were not created). Even better would be to use the `tempfile` module to make a temporary directory for the files rather than writing them the working directory of the test.